apps/nova/src/components/TutorProfile/EditRating.tsx

```diff
@@ -0,0 +1,25 @@
+import { Dialog, useFormatMessage } from "@litespace/luna";
+import React from "react";
+import RateForm from "./RateForm";
+import { IRating } from "@litespace/types";
+
+type EditRatingProps = {
+  open: boolean;
+  close: () => void;
```

Comment:
Please use `Void` custom types. 

```suggestion
  close: Void;
```